main_2024.py

```diff
@@ -49,15 +49,17 @@ def main() -> int:
     try:
         QUEUE_MAX_SIZE = config["queue_max_size"]
 
+        LOG_DIRECTORY_PATH = config["log_directory_path"]
+
         VIDEO_INPUT_CAMERA_NAME = config["video_input"]["camera_name"]
         VIDEO_INPUT_WORKER_PERIOD = config["video_input"]["worker_period"]
-        VIDEO_INPUT_SAVE_PREFIX = config["video_input"]["save_prefix"]
+        VIDEO_INPUT_SAVE_PREFIX = f"{LOG_DIRECTORY_PATH}/{config['video_input']['save_prefix']}"
```

Comment:
To avoid the nested quotation marks, perhaps something like this:
```py
VIDEO_INPUT_SAVE_PREFIX = str(pathlib.Path(LOG_DIRECTORY_PATH, config["video_input"]["save_prefix"]))
```